src/touchstone/compare.py

```diff
@@ -141,8 +141,8 @@ def main(args):
     # Get indices from metadata map
     metadata_search_map = benchmark_instance.search_map_metadata
     metadata_dict = {}
-    if args.tolerancy_rules and len(args.uuid) < 2:
-        logger.critical("At least two uuids are required when tolerancy-rules flag is passed")
+    if args.tolerancy_rules and (len(args.uuid) < 2 or len(args.uuid) > 2):
+        logger.critical("Two uuids exactly are required when tolerancy-rules flag is passed")
```

Comment:
This is a confusing log message. 

"No more than two UUIDs are allowed when passing the `tolerancy-rules` flag."

Reply:
here we need exactly 2 uuid's , not more not less